.travis.yml

```diff
@@ -0,0 +1,26 @@
+dist: trusty
```

Comment:
Did you choose 'trusty' on purpose here?  This should be fine, we've just been using a debian image in our test suite.

Reply:
https://docs.travis-ci.com/user/reference/overview/#virtualization-environments

I try to upgrade it to xenial or bionic, but this one works well, just let me know and I can give it a shot.

Reply:
I think this is good for now.  We can change it later if we see it causes strife.